tests/test_distro.py

```diff
@@ -138,11 +138,10 @@ def _setup_for_distro(self, distro_root: str) -> None:
 class TestOSRelease:
     def setup_method(self, test_method: FunctionType) -> None:
         dist = test_method.__name__.split("_")[1]
-        os_release = os.path.join(DISTROS_DIR, dist, "etc", "os-release")
         self.distro = distro.LinuxDistribution(
             include_lsb=False,
-            os_release_file=os_release,
             distro_release_file="path-to-non-existing-file",
+            root_dir=os.path.join(DISTROS_DIR, dist),
```

Comment:
The intentions (and need) for the changes to `setup_method` above are not clear to me. Are they needed? Could you elaborate?

Reply:
It appeared that `TestOSRelease` test cases **were not** "chroot-ed" in their corresponding directories... Now that we open a file under `/etc` it became an issue (host's `/etc/debian_version` was read).
I've had to set `root_dir` to workaround this as well :confused:

Reply:
The access to `self.etc_dir`, I see! :+1: